seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -443,7 +443,7 @@
       "translations": {
         "de": {
           "description": [
-            "Denk daran, dass du einen Kommentar mit <code>&#60;!--</code> anfangen und mit <code>--&#62;</code> wieder beenden werden.",
+            "Denk daran, dass du einen Kommentar mit <code>&#60;!--</code> anfangen und mit <code>--&#62;</code> wieder beendest.",
```

Comment:
"dass ein Kommentar mit ... beginnt und mit ... endet" Maybe something along the lines?